src/jit.c

```diff
@@ -56,7 +56,8 @@
 #define JIT_OP_MOD_REG (JIT_CLS_ALU | JIT_SRC_REG | 0x90)
 
 #define STACK_SIZE 512
-#define MAX_INSNS 1024
+#define MAX_JUMPS 1024
```

Comment:
Can you show empirical adjustments for both `MAX_JUMPS` and `MAX_INSNS`. I wonder how they can be calculated.

Reply:
The `MAX_JUMPS` means the number of host branch instructions within a chained block collection need to be resolved.

Reply:
I modified `MAX_INSNS` into `MAX_BLOCKS` because it refers to the total number of basic blocks within our code cache. Considering that our block cache size is 4096 and some blocks may be replaced, I set MAX_BLOCKS to 8192.